src/index.ts

```diff
@@ -327,17 +388,41 @@ export async function startAnalysis<
 
     const platform = platformApi(platformOptions);
 
+    const device = {
+        platform: analysisOptions.platform,
+        runTarget: analysisOptions.runTarget,
+        osVersion: await platform.getDeviceAttribute('osVersion'),
+        osBuild: await platform.getDeviceAttribute('osBuild'),
+        manufacturer: await platform.getDeviceAttribute('manufacturer'),
+        ...(platform.target.platform === 'android' && {
+            model: await (platform as unknown as PlatformApi<'android', RunTarget, [], 'wireguard'>).getDeviceAttribute(
+                'model'
+            ),
+        }),
+        modelCodeName: await platform.getDeviceAttribute('modelCodeName'),
+    };
```

Comment:
This doesn't work if CA is starting the emulator. We'll try to communicate via adb with a device that isn't started yet (that only happens in `ensureDevice()`.